### DISCUSSION TITLE
Prettier Configuration: spaces instead of tabs

### DISCUSSION THREAD

**Discussion by andrashee** [socialincome-san/public#121] [Archived Discussions]
The [prettier](https://github.com/socialincome-san/public/blob/a965330d4042881f84f7252caf4ac1b3182d35f9/.prettierrc#L4) configuration which we are currently using is using 2 tabs for code.

I find this very unhandy to read, especially in PRs. E.g.
 
<img width="1651" alt="Screenshot 2022-10-07 at 15 41 20" src="https://user-images.githubusercontent.com/1204548/194568552-b12a2445-5ae8-4a5d-b367-5e9aaff49d4d.png">

Shall we switch back to the default 2 spaces? Wdyt @socialincome-san/admins-public-repo?

**Comment by renestalder** [comment]
@andrashee Change your tab size setting on GitHub.

https://github.com/settings/appearance

That's the beauty of tabs: Everyone can use their own view settings.

**Reply by renestalder** [reply]
I don't care that much, btw. whether we use tabs or spaces. I just know that tabs are more accessible for the reason that everyone can apply their own view settings to them, and also that the code base is smaller when using tabs. Apart from that, I always used spaces and only recently switched to tabs when I found out there is an actual difference in accessibility. So, I use 2 spaces for one tab, but someone else might prefer 4 spaces for one tab.

https://adamtuttle.codes/blog/2021/tabs-vs-spaces-its-an-accessibility-issue/

**Reply by andrashee** [reply]
Ah splendid, wasn't aware! That resolves the issue for me 👍 Thanks for the quick reaction.
